src/common/boardwrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import { BoardProps } from "boardgame.io/dist/types/packages/react";
+
+export function boardWrapper(board : any, description: any) { // TODO: solve types with BoardProps<MyGameState>
```

Comment:
I am not sure if we should always have a non-any type: for example for board it seems nice that depending on the game it can be array, object, single number etc (but I am new to typescript...)

Reply:
I may have misunderstood the issue at hand, but I think it is possible using non-any in most, if not all, type declarations and functions. For our goals, it should be possible.

TypeScript has a robust type system which allows us to plug any game's state object as a part of our bigger state.

e.g. `BigState<MyBoardState>` could be used to provide a list of attributes from BigState (I do not have the code at my disposal, but let's say x.description), but x.state could refer to a type specified by the game implementation.

`type BigState<T> = { state: T; description: string; }`

Reply:
This should be great. I'll add an issue for this.